chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -298,8 +298,8 @@
   end
 
   if node[:platform_family] == "suse"
-    ## make sure the repos are properly setup
-    repos = Provisioner::Repositories.get_repos(node["platform"], node["platform_version"])
+    # make sure the repos are properly setup
+    repos = Provisioner::Repositories.get_repos(node[:platform], node[:platform_version], node[:kernel][:machine])
```

Comment:
Metrics/LineLength: Line is too long. [114/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)